_includes/manuals/1.12/3.2.2_installer_options.md

```diff
@@ -1009,7 +1009,7 @@ For example the hash `{show_diff => true, stringify_facts => false}` for `--pupp
       </tr>
       <tr>
         <td style='white-space:nowrap'>--foreman-proxy-virsh-network</td>
-        <td>Network for virsh DNS/DHCP provider</td>
+        <td>Network for virsh DNS/DHCP provider (deprecated for 1.12+)</td>
```

Comment:
This change isn't in the Puppet module and in fact, the parameter's been renamed. I'd just leave this out until the doc is synced correctly.